eth_contracts/contracts/Battle.sol

```diff
@@ -2,4 +2,50 @@ pragma solidity ^0.4.17;
 
 contract Battle {
 	
+}
+
+contract BattleGroups {
+	struct BattleGroup {
+		// The timestamp from the block when the battlegroup was created
+		uint64 creationTime;
+
+		// Address of owner of BattleGroup
+		address owner;
+
+		// Cards which belong to the BattleGroup
+		uint256[3] cards;
```

Comment:
Consider going to 5, have null space and a variable which tracks how many cards we're accepting to parse through (leave rest null/empty). I.e. uint16 teamSize;